nixos-x86_64.json

```diff
@@ -52,6 +52,33 @@
           "1024"
         ]
       ]
+    },
+    {
+      "boot_wait": "40s",
+      "boot_command": [
+        "echo http://{{.HTTPIP}}:{{.HTTPPort}} > .packer_http<enter>",
+        "mkdir -m 0700 .ssh<enter>",
+        "curl $(cat .packer_http)/install_rsa.pub > .ssh/authorized_keys<enter><wait>",
+        "nixos-rebuild switch<enter><wait>",
```

Comment:
is this line necessary? It's not used for the other targets

Reply:
Just remove that line and tested ... It is actually needed because nixos has to be rebuild with  `virtualisation.hypervGuest.enable = true;'`  otherwise the host never get the IP back and ssh callback will never fire. That means that I need the `sed` call.

Can you think of another way to make this work ?

For your information, even when the hyperv packer images succeeded, I have never managed to get it work with vagrant. For some reason the X server will fail to start up ...

At the end, `hyperv` has been a gigantic piece of wasted time on my side :-(

Reply:
Ouch, sorry to hear that. I think the best way would be to change the generation of images to be Nix-driven instead of Packer-driven. But that will take some more work.

Reply:
The least effort would to cave in and install VirtualBox I'm afraid. I also think that Vagrant mainly works well with VirtualBox, all the other targets are less tested.

Reply:
Ok no problem. I don't mind closing the PR and keeping the branch for documentation purpose.

As a note, what motivates me to investigate other hypervisors is the need for nested virtualization (not supported by Virtualbox on Intel).